solc/CommandLineInterface.cpp

```diff
@@ -1122,7 +1158,11 @@ void CommandLineInterface::assembleYul(yul::YulStack::Language _language, yul::Y
 			else
 				serr() << "No binary representation found." << endl;
 		}
-
+		if (m_options.compiler.outputs.astCompactJson)
+		{
+			sout() << "IR JSON AST (compact format):" <<  endl <<  endl;
+			sout() << util::jsonPrint(stack.printJson(), util::JsonFormat{ util::JsonFormat::Compact }) << endl;
+		}
```

Comment:
Looks like there are zero tests covering the three newly added CLI options. Can you add a simple command-line test that uses them?